src/dev/typescript/projects.ts

```diff
@@ -35,6 +35,10 @@ export const PROJECTS = [
     name: 'apm/cypress',
     disableTypeCheck: true,
   }),
+  new Project(resolve(REPO_ROOT, 'x-pack/plugins/apm/new_e2e/tsconfig.json'), {
+    name: 'apm/e2e_cypress',
```

Comment:
Let's try to have a single name for the test suite, eg. "new_e2e" or as I suggested above: "ftr_e2e"

```suggestion
    name: 'apm/ftr_e2e',
```